provisioning/lib/helpers

```diff
@@ -0,0 +1,26 @@
+# Usage: error_out <message> <error_code>
+error_out() {
+  echo "ERROR: ${1}"
+  exit $2
+}
+
+# Usage: get_hostname <ip address>
```

Comment:
@etsauer I didn't notice this when I left the comment yesterday, but the \< and \> has to be escaped in github commentary fields and forgot to do so. I updated the above comment and I think it may make more sense now.